index.html

```diff
@@ -65,6 +65,80 @@ <h2>Introduction</h2>
       TODO(henbos): Fill in.
     </p>
   </section>
+  <section id="*">
+    <h3>
+      <dfn>RTCRtpReceiver</dfn> extensions
+    </h3>
+    <p>
+      Let <code>RTCRtpReceiver</code> objects have an
+      <dfn>[[\ReceiverJitterBufferDelayHint]]</dfn> internal slot initially
+      initialized to <code>null</code>.
+    </p>
+    <pre class="idl">partial interface RTCRtpReceiver {
+  attribute double? jitterBufferDelayHint;
+};</pre>
+    <section>
+      <h2>Attributes</h2>
+      <dl data-link-for="RTCRtpReceiver" data-dfn-for="RTCRtpReceiver"
+      class="attributes">
+        <dt><code><dfn>jitterBufferDelayHint</dfn></code> of type <span class=
+        "idlAttrType"><a>double</a></span>, nullable</dt>
+        <dd>
+          <p>This attribute gives the application the ability to affect the
+          length of the jitter buffer using a <i>hint</i>. This allows the
+          application to influence the trafeoffs between having a longer or
+          shorter jitter buffer. The longer the buffer, the more chance there
+          is that packets arrive in time for playout, but the higher the delay
+          is. The shorter the buffer, the higher the risk of stuttering, but
+          the shorter the playout delay. For interactive applications, low
+          delay may be important, wheras for passive applications a
+          stutter-free experience may be more important.</p>
+          <p>Note that the delay is only a hint; a soft target. The underlying
+          congestion control mechanisms employed by user agensts are outside
+          the scope of this document.</p>
+          <p>The user agent SHOULD let other factors, such as performance,
+          network conditions and user experience, take higher presidence in
+          the calculation of the actual jitter buffer delay, than the current
+          hint set by the application. How the user agent does or does not
+          take the hint into account under different circumstances is up to
+          the user agent.</p>
```

Comment:
This is hardly a description that can lead to interoperable implementation....

I'd suggest a more normative approach, while still giving leeway, something like:

The JitterBufferHint is a hint about how much additional delay the system can add, above the minimum delays implied by decoding and playout, without the application considering it to be a problem. This may be a small number of milliseconds for interactive applications, while it can be multiple seconds for passive playback applications. 

When set, the user agent SHOULD use tools at its disposal, such as inserting or removing samples, to make sure the delay is reasonably close to the given value after some amount of time has passed if no disruptions occur. The user agent SHOULD NOT delay the start of media playout by any appreciable amount.